release/models/system/openconfig-system.yang

```diff
@@ -266,6 +272,17 @@ module openconfig-system {
         information such as the current system date and time, uptime,
         last login timestamp, etc.";
     }
+
+    leaf my-mac-address {
```

Comment:
Should we call this `local-mac-address` or similar? I'm not sure whether we ever refer to a device owning things with `my` anywhere in the models.

Reply:
+1 to above comment - if this is only ever singular (as previously mentioned) and system wide just simply `/system/config/mac-address` or `/system/config/local-mac-address` should suffice.  Would recommend against first person pronoun here.

Reply:
Addressed all of these in new commit.